app/models/piece.rb

```diff
@@ -1,12 +1,30 @@
 require 'pry'
+
+class Move
+  def new(x, y)
+    @x = x
+    @y = y
+  end
+
+  attr_reader :x
+
+  attr_reader :y
+end
```

Comment:
You could use a `struct` for this instead. http://blog.steveklabnik.com/posts/2012-09-01-random-ruby-tricks--struct-new